plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/procedure/VacuumProcedure.java

```diff
@@ -183,6 +183,7 @@ private void doVacuum(
         List<Long> recentVersions = transactionLogAccess.getPastTableVersions(fileSystem, transactionLogDir, threshold, tableSnapshot.getVersion());
         Set<String> retainedPaths = Stream.concat(
                         transactionLogAccess.getActiveFiles(tableSnapshot, session).stream()
+                                // paths can be absolute as well in case of shallow-cloned tables, but they shouldn't and aren't being deleted as part of vacuum
```

Comment:
Please add test about `vacuum` procedure with shallow-cloned table.

Reply:
@ebyhr I have added the test in `TestDeltaLakeCloneTableCompatibility` itself, please let me know If I need to separate this.